--- libgnomeprint-2.18.8.org/libgnomeprint/gnome-font-face.c	2010-02-09 13:32:51.000000000 +0100
+++ libgnomeprint-2.18.8/libgnomeprint/gnome-font-face.c	2025-07-19 17:22:11.765065918 +0200
@@ -1003,7 +1003,7 @@ typedef struct {
 	gdouble * t;
 } GFFT2OutlineData;
 
-static int gfft2_move_to (FT_Vector * to, void * user)
+static int gfft2_move_to (const FT_Vector * to, void * user)
 {
 	GFFT2OutlineData * od;
 	ArtPoint p;
@@ -1025,7 +1025,7 @@ static int gfft2_move_to (FT_Vector * to
 	return 0;
 }
 
-static int gfft2_line_to (FT_Vector * to, void * user)
+static int gfft2_line_to (const FT_Vector * to, void * user)
 {
 	GFFT2OutlineData * od;
 	ArtBpath * s;
@@ -1048,7 +1048,7 @@ static int gfft2_line_to (FT_Vector * to
 	return 0;
 }
 
-static int gfft2_conic_to (FT_Vector * control, FT_Vector * to, void * user)
+static int gfft2_conic_to (const FT_Vector * control, const FT_Vector * to, void * user)
 {
 	GFFT2OutlineData * od;
 	ArtBpath * s, * e;
@@ -1076,7 +1076,7 @@ static int gfft2_conic_to (FT_Vector * c
 	return 0;
 }
 
-static int gfft2_cubic_to (FT_Vector * control1, FT_Vector * control2, FT_Vector * to, void * user)
+static int gfft2_cubic_to (const FT_Vector * control1, const FT_Vector * control2, const FT_Vector * to, void * user)
 {
 	GFFT2OutlineData * od;
 
--- libgnomeprint-2.18.8/tests/pango.c~	2025-07-19 17:30:32.000000000 +0200
+++ libgnomeprint-2.18.8/tests/pango.c	2025-07-19 17:31:10.875121316 +0200
@@ -158,7 +158,7 @@ main (int argc, char * argv[])
 		usage ("Input file not specified");
 	input_file = args[0];
 
-	if (!g_file_get_contents (input_file, &text, &len, &error))
+	if (!g_file_get_contents (input_file, &text, (gsize *)&len, &error))
 		fail ("%s\n", error->message);
 	if (!g_utf8_validate (text, len, NULL))
 		fail ("Text is not valid UTF-8");
