diff -urN aa/libgnomeprint-2.6.1/configure.in libgnomeprint-2.6.1/configure.in
--- aa/libgnomeprint-2.6.1/configure.in	2004-04-19 15:00:32.000000000 +0200
+++ libgnomeprint-2.6.1/configure.in	2004-04-21 23:32:11.916199104 +0200
@@ -243,16 +243,6 @@
 dnl END: LIBGNOMEPRINT_LIBS & CFLAGS
 dnl ================================
 
-dnl =================================================
-dnl Get the loadable module extension for this system
-dnl =================================================
-AC_LTDL_SHLIBEXT
-LTDL_SHLIBEXT=$libltdl_cv_shlibext
-AC_SUBST(LTDL_SHLIBEXT)
-dnl ======================================================
-dnl END: Get the loadable module extension for this system
-dnl ======================================================
-
 dnl =======================================
 dnl Whether to install the metadata printer
 dnl =======================================
diff -urN aa/libgnomeprint-2.6.1/libgnomeprint/gpa/gpa-printer.c libgnomeprint-2.6.1/libgnomeprint/gpa/gpa-printer.c
--- aa/libgnomeprint-2.6.1/libgnomeprint/gpa/gpa-printer.c	2003-11-26 21:32:36.000000000 +0100
+++ libgnomeprint-2.6.1/libgnomeprint/gpa/gpa-printer.c	2004-04-21 23:33:07.951680416 +0200
@@ -32,6 +32,7 @@
 #include <dirent.h>
 
 #include <gmodule.h>
+#include <glibconfig.h>
 #include <libxml/parser.h>
 #include <libxml/xmlmemory.h>
 
@@ -386,7 +387,7 @@
 {
 	struct dirent *entry;
 	DIR *dir;
-	gint ext_len = strlen (LTDL_SHLIB_EXT);
+	gint ext_len = strlen (G_MODULE_SUFFIX);
 	g_assert (ext_len > 0);
 
 	if (!g_module_supported ()) {
@@ -410,7 +411,7 @@
 		if (len < ext_len + 2) /* 2 = one char + 1 for '.'*/
 			continue;
 
-		if (strcmp (entry->d_name + len - ext_len, LTDL_SHLIB_EXT))
+		if (strcmp (entry->d_name + len - ext_len, G_MODULE_SUFFIX))
 			continue;
 		
 		path = g_build_filename (dir_path, entry->d_name, NULL);
